public/app/features/plugins/components/AppRootPage.tsx

```diff
@@ -157,4 +157,14 @@ export function getAppPluginPageError(meta: AppPluginMeta) {
   return null;
 }
 
+function convertLocationSearchToObject(locationSearch: string) {
```

Comment:
nit: I think we have a utility function available in runtime [here](https://github.com/grafana/grafana/blob/main/packages/grafana-runtime/src/services/LocationService.ts#L125-L140) that does a similar job. However this feels easier to reason with so I'm wondering if in the future we should replace the util in runtime with this implementation. WDYT?

Reply:
Sounds like a good suggestion bro! 👍